docs/yaml.md

```diff
@@ -68,6 +68,19 @@ build:
     - "git+https://github.com/m-bain/whisperX.git"
 ```
 
+You can also pin Python package installations to a specific git commit:
+
+```yaml
+build:
+  system_packages:
+    - "git"
+  python_packages:
+    - "git+https://github.com/m-bain/whisperX.git@f2da2f858e99e4211fe4f64b5f2938b007827e17"
```

Comment:
This example format didn't work for me, I think it needs to be in the format:

`- cog-model-helpers @ git+https://github.com/fofr/cog-model-helpers.git@19b0a2a2ad9f4e464c3573717259915f8250e594`

Reply:
Oh i see. I didn't test it. Is that for any git-based installation? Or just when you're specifying the SHA? (In other words, is the existing doc here also maybe wrong?)

Reply:
The existing pattern still works, but you cannot append a SHA to it

Reply:
Updated!